### DISCUSSION TITLE
[Submod] Interact Zone (Beta)

### DISCUSSION THREAD

**Discussion by Lu1508** [Monika-After-Story/MonikaModDev#9401] [general - submod / external content]
This is a small submod that let you interact with Monika by clicking on her

![Screenshot from 2022-07-26 23-04-06](https://user-images.githubusercontent.com/84401880/181144762-f845db0a-045b-462a-b1b0-3743721e44b0.png)
![Screenshot from 2022-07-26 23-03-40](https://user-images.githubusercontent.com/84401880/181144771-03b84d0f-03e6-4dc2-9da4-4f842181d9fe.png)

READ THIS:
- The interactive parts are: Head - Nose - Cheeks - Hair - Hands - Stomach
- It's based on some of the dialogues of "Extra Everything" submod, so I did the code there
- The buttons are invisible to make it look realistic, but you will know your mouse is on the correct place when it makes a sound
- I wanted her to act like she is in her waiting mode, but, with the lean of her body and the change of her hands' position, I decided to make her static, waiting to click on her (that's why is a beta)
- You can ask her if you can touch her (it's optional)
- It's recommended to use it with the default zoom level (the buttons don't change size with her)
- Download the files and paste them in the game folder (if you don't have the "Extra Everything" submod, here is the link: https://drive.google.com/file/d/1iCzq1J76a1x-qUm-3gnvUYKpErqvaTEC/view)

Link: https://drive.google.com/drive/folders/1vCNBj91FI4fbCJe0-ghLB2c6LSJ7ZQOT?usp=sharing

**Comment by Waffles201302** [comment]
Im having trouble and kinda confused with downloaidng and pasting in game folder (i have the extra everything mod also)

**Reply by Waffles201302** [reply]
Oh ok

**Reply by charleseyler** [reply]
hiii can you explain to me how to download it because I can't seem to figure it out

**Reply by destiny6destroyed** [reply]
if I can guess, you have to paste the files inside the game folder (most of the submods work that way)

**Reply by Waffles201302** [reply]
nah the submod doesnt work

**Reply by AbacaxiV** [reply]
it doesn't work. At all

**Reply by destiny6destroyed** [reply]
yeah, and it looks like the creator disappeared

I suggest you to use this submod instead (it has the interact thing and more cool stuff): https://github.com/zer0fixer/MAS-Extraplus

**Comment by bunniilove7** [comment]
hii, the extra everything link doesnt work anymore :(

**Reply by destiny6destroyed** [reply]
just checked and the extra everything submod is inside the submods folder of this submod creator

**Reply by dinguds** [reply]
pardon? i appreciate the help, but i dont understand :P

**Reply by destiny6destroyed** [reply]
> pardon? i appreciate the help, but i dont understand :P

nevermind, even if it still inside, there are a lot of complains that it doesn't work anyway and looks like the creator disappeared

I suggest you to use this submod instead (it has the interact thing and more cool stuff): https://github.com/zer0fixer/MAS-Extraplus

**Comment by destiny6destroyed** [comment]
the script-ch30 file has some errors you need to delete or it doesn't let you run MAS at all

```
- python:
        game_unlock_db = {
            "chess": "mas_unlock_chess",
            mas_games.HANGMAN_NAME: "mas_unlock_hangman",
            "piano": "mas_unlock_piano",
            "nou": "mas_reaction_gift_noudeck"
        }
        mas_unlockGame("pong") 

        for game_name, game_startlabel in game_unlock_db.iteritems():
            
            if mas_getEVL_shown_count(game_startlabel) > 0:
                mas_unlockGame(game_name)

- $ store.mas_island_event.advanceProgression()

- store.mas_island_event.advanceProgression()
```


leaving that aside, the submod works quite well, good job
